flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/CommitSummary.java

```diff
@@ -19,13 +19,14 @@
 package org.apache.iceberg.flink.sink;
 
 import java.util.Arrays;
+import java.util.List;
 import java.util.NavigableMap;
 import java.util.concurrent.atomic.AtomicLong;
 import org.apache.iceberg.io.WriteResult;
 import org.apache.iceberg.relocated.com.google.common.base.MoreObjects;
 import org.apache.iceberg.util.ScanTaskUtil;
 
-class CommitSummary {
+public class CommitSummary {
```

Comment:
Is this still an `@Internal` class?

Reply:
Are you suggesting to annotate with `@Internal` because of the visibility change?

Reply:
Yes

Reply:
These are Flink annotations though, not sure how applicable they are in the context of Iceberg, but I've added them.